### PR TITLE
lintcode: 57-3Sum

### PR DESCRIPTION
**Pull request by wen-wayne-shih**
- lintcode: 57-3Sum

### CODE REVIEW

File: lintcode/two_pointers/57-3Sum/solution.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        for i in range(len(numbers) - 2):
+            if i > 0 and numbers[i] == numbers[i - 1]:
+                continue
+            self._two_sum_unique(numbers, i, results)
+
+        return results
+
+    def _two_sum_unique(self, nums, first_idx, results):
+        target = -nums[first_idx]
+        l, r = first_idx + 1, len(nums) - 1
```
Comment (W-Shih) [inline lintcode/two_pointers/57-3Sum/solution.py:36]:
```suggestion
    def _two_sum_unique(self, nums, start_idx, end_idx, target, results):

```

**Review by W-Shih — CHANGES_REQUESTED** [review]
must be addressed before merging.

File: lintcode/two_pointers/57-3Sum/solution.py (hunk 1 above)
Comment (wen-wayne-shih) [reply lintcode/two_pointers/57-3Sum/solution.py]: Resolved as per suggestion

**Review by W-Shih — CHANGES_REQUESTED** [review]
squash commits before merging

**Review by W-Shih — APPROVED** [review]

**Pull request merged**